statsmodels/stats/rates.py

```diff
@@ -30,17 +30,17 @@ def test_poisson_2indep(count1, exposure1, count2, exposure2, ratio_null=1,
 
     Parameters
     ----------
-    count1: int
+    count1 : int
```

Comment:
those changes will cause merge conflicts with the docs that I'm working on right now
proportion, rates, oneway, meta_analysis, ...

Reply:
might cause*

Reply:
I checked, it is a very simple rebase that is already needed.